isolate_example/lib/page_two.dart

```diff
@@ -157,17 +164,28 @@ class RunningList extends StatelessWidget {
 
     return DecoratedBox(
       decoration: BoxDecoration(
-        color: (controller.running == true && controller.paused == false)
-            ? Colors.lightGreenAccent
-            : Colors.deepOrangeAccent,
+        color: Colors.grey[200],
       ),
       child: new ListView.builder(
         itemCount: sums.length,
         itemBuilder: (context, index) {
-          return ListTile(
-            title: Text((sums.length - index).toString() +
-                '. ' +
-                sums[index].toString()),
+          return Column(
+            children: <Widget>[
```

Comment:
Don't need the `&lt;Widget&gt;` here, since the list has widgets of different types.